src/style.css

```
@@ -1,3 +1,89 @@
 body {
   margin: 0;
+  --main-color: #00acdc;
+  --back-color: #fff;
+  --text-color: #000;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  font-family: "Roboto", sans-serif;
+  background-color: var(--back-color);
+  width: 100%;
+  max-width: 1200px;
+}
+
+.logo {
+  height: 40px;
+  margin: 10px 0 10px 50px;
+}
+
+nav.list {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  margin-right: 50px;
+}
+
+ul.navigation {
+  list-style: none;
+  display: flex;
+  justify-content: space-around;
+  padding-inline-start: 0;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+li.nav-item {
+  padding-left: 20px;
+}
+
+li.is-active {
+  padding-left: 0;
+}
```

to make outer indent you should use margin
```suggestion
.nav__item:not(:last-child) {
  margin-right: 20px;
}
```